### ISSUE TITLE
Search bar on home page does not have label

### ISSUE THREAD

**Issue by SukhsimranpreetSekhon** [CDH-Studio/UpSkill#945]
**Describe the bug**
- The search bar on the home page does not have a label.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to the home page after login.
2. See Search bar.

**Expected behaviour**
- Need to add an appropriate label for the search bar that is English and French translateable.

**Screenshots**

**Additional context**
